cache/slru_cache.cpp

```diff
@@ -395,13 +395,12 @@ int slru_cache_t::lookup(const unsigned char *id, dnet_net_state *st, dnet_cmd *
 	data_t* it = m_treap.find(id);
 	TIMER_STOP("lookup.find");
 
-	dnet_time timestamp;
-	memset(&timestamp, 0, sizeof(timestamp));
-
-	if (it) {
-		timestamp = it->timestamp();
+	if (!it) {
+		return -ENOTSUP;
```

Comment:
This completely breaks previous logic.
Also, this change is far out of logging changes this pull request is about.